chain/transaction.go

```diff
@@ -144,12 +144,12 @@ func (t *Transaction) StateKeys(sm StateManager) (set.Set[string], error) {
 	if t.WarpMessage != nil {
 		p := sm.IncomingWarpKeyPrefix(t.WarpMessage.SourceChainID, t.warpID)
 		k := keys.EncodeChunks(p, MaxIncomingWarpChunks)
-		stateKeys.Add(string(k))
+		stateKeys.Add(state.NewKey(string(k), state.Read, state.Write))
```

Comment:
Should just be write

Reply:
I ran the `panic` and it showed that GetValue was being called here: https://github.com/ava-labs/hypersdk/blob/797d38ae84c8d7dca027a6b1d3e875e49218082e/chain/transaction.go#L376-L379

Reply:
That's my bad, this should be RW (we use this to check for duplicates).